src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
Whether the opponent has to capture on his next move does not say much about whether our move was a capture, so I am just distinguishing one more "type" of move there. Can you explain what is so awfully complicated about this?